src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
```suggestion
public interface AreaCalculator {
```

Обычно названия классов не начинаются с глаголов